website/src/controllers/groups.js

```diff
@@ -443,6 +443,42 @@ api.likeChatMessage = function(req, res, next) {
   })
 }
 
+/**
+ * Get group members
+ */
+api.members = function(req, res, next) {
+  var user = res.locals.user;
+  var gid = req.params.gid;
+  var limit = req.body.limit;
+  var offset = req.body.offset;
+
+  var q = (gid == 'party')
+    ? Group.findOne({type: 'party', members: {'$in': [user._id]}})
+    : Group.findOne({$or:[
+        {_id:gid, privacy:'public'},
+        {_id:gid, privacy:'private', members: {$in:[user._id]}} // if the group is private, only return if they have access
```

Comment:
parties always have a privacy type of `'private'`, couldn't we just adjust the query based on that?